application/src/main/kotlin/org/gxf/soapbridge/configuration/properties/SoapConfigurationProperties.kt

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: Copyright Contributors to the GXF project
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.gxf.soapbridge.configuration.properties
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties("soap")
+class SoapConfigurationProperties(
+    val hostnameVerificationStrategy: HostnameVerificationStrategy,
+    /**
+     * Maximum number of seconds this {@link SoapEndpoint} will wait for a response from the other end
+     * before terminating the connection with the client application.
+     */
+    val timeOut: Int,
+    /**
+     * Time outs for specific functions.
```

Comment:
"timeout" en "time-out" zijn correct maar "time out" niet :D
Is nu ook een beetje incosistent want hier boven i O een hoofdletter en hier onder niet.